deployment/services/public-graphql-api-gateway.ts

```diff
@@ -52,6 +55,7 @@ export function deployPublicGraphQLAPIGateway(args: {
   const configMap = new kx.ConfigMap('public-graphql-api-gateway-config', {
     data: {
       'gateway.config.ts': gwConfigFile,
+      'setup-telemetry.ts': gwTelemetryFile,
```

Comment:
Can be, but doesnt have to be a separate file. Just an fyi, whatever feels cleaner to you.